src/plugins/irc-events/error.js

```diff
@@ -44,10 +44,22 @@ module.exports = function(irc, network) {
 	});
 
 	irc.on("nick in use", function(data) {
+		let message = data.nick + ": " + (data.reason || "Nickname is already in use.");
+
+		if (irc.connection.registered === false && !Helper.config.public) {
+			message += " An attempt to use it will be made when this nick quits.";
```

Comment:
`quits` -> `becomes available`?

Reply:
`quits` at least reflects when TL will actually attempt to use that nick, if you say `becomes available`, people might expect a little more magic behind that phrasing.

Reply:
Yeah, `quit` is probably too jargon.

Maybe...

```suggestion
			message += " An attempt to use it will be made when this nick is no longer in use.";
```

Reply:
`quit` is an action you can see in channels, so I don't think it's too jargon. I'd rather have this sentence attached to an actual action (which hopefully prevents people from questioning on what use/available means).

If we update keep nick logic to also work in other scenarios like nick changes, then we can certainly change this wording ☺

Reply:
I'm not convinced tbh, but I won't fight it. It's not something that'll come up much anwyay.